itest/lnd_open_channel_test.go

```diff
@@ -822,3 +822,53 @@ func testSimpleTaprootChannelActivation(ht *lntest.HarnessTest) {
 	// Our test is done and Alice closes her channel to Bob.
 	ht.CloseChannel(alice, chanPoint)
 }
+
+// testOpenChannelLockedBalance tests that when a funding reservation is
+// made for opening a channel, the balance of the required outputs shows
+// up as locked balance in the WalletBalance response.
+func testOpenChannelLockedBalance(ht *lntest.HarnessTest) {
+	// Before we start the test, we'll ensure both sides are connected so
+	// the funding flow can be properly executed.
+	alice := ht.Alice
+	bob := ht.Bob
+	ht.EnsureConnected(alice, bob)
```

Comment:
nit: this should have already been asserted by the framework itself.

Reply:
Removed this line.